docs/src/consts.ts

```diff
@@ -53,6 +53,9 @@ export const SIDEBAR: Sidebar = {
       { text: "Introduction", link: "en/introduction" },
       { text: "Installation", link: "en/installation" },
     ],
+    // Usage: [
```

Comment:
@hqasmei is this needed?

Reply:
Had it there as reference to test the og stuff. Will removed it.